### PR TITLE
Feat/vite react

### PR DESCRIPTION
**Pull request by trungbach**
What is purpose of the changes?

 - Use Vite as dev server instead of react-app-rewired for a better development experience.

### CODE REVIEW

File: src/pages/Pools/components/AddLiquidityModal/AddLiquidityModal.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
+import CloseIcon from 'assets/icons/ic_close_modal.svg?react';
+import ImgPairPath from 'assets/images/pair_path.svg?react';
```
Comment (quangdz1704) [inline src/pages/Pools/components/AddLiquidityModal/AddLiquidityModal.tsx:2]: img import

File: src/pages/Pools/components/Header/Header.tsx [hunk 2]
```diff
@@ -1,5 +1,11 @@
 import { ExecuteInstruction } from '@cosmjs/cosmwasm-stargate';
 import { CW20_DECIMALS, ORAI, ORAI_INFO, USDT_CONTRACT, toDecimal, toDisplay } from '@oraichain/oraidex-common';
+import UpIcon from 'assets/icons/up-arrow.svg?react';
```
Comment (quangdz1704) [inline src/pages/Pools/components/Header/Header.tsx]: maybe it is unused import?

File: src/pages/Pools/components/ListPoolMobile/ListPoolMobile.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { TokenItemType } from '@oraichain/oraidex-common';
-import NoDataSvg from 'assets/images/NoDataPool.svg';
-import NoDataLightSvg from 'assets/images/NoDataPoolLight.svg';
+import NoDataSvg from 'assets/images/NoDataPool.svg?react';
```
Comment (quangdz1704) [inline src/pages/Pools/components/ListPoolMobile/ListPoolMobile.tsx:2]: img import

File: src/pages/Pools/components/MyPoolInfo/MyPoolInfo.tsx [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
-import { ReactComponent as WithdrawIcon } from 'assets/icons/ic_withdraw.svg';
-import { ReactComponent as WithdrawLightIcon } from 'assets/icons/ic_withdraw_light.svg';
-import img_coin from 'assets/images/img_staked.svg';
+import DepositIcon from 'assets/icons/ic_deposit.svg?react';
+import StakingIcon from 'assets/icons/ic_stake.svg?react';
+import UnstakeIcon from 'assets/icons/ic_unstake.svg?react';
+import UnstakeLightIcon from 'assets/icons/ic_unstake_light.svg?react';
+import WithdrawIcon from 'assets/icons/ic_withdraw.svg?react';
+import WithdrawLightIcon from 'assets/icons/ic_withdraw_light.svg?react';
+import img_coin from 'assets/images/img_staked.svg?react';
```
Comment (quangdz1704) [inline src/pages/Pools/components/MyPoolInfo/MyPoolInfo.tsx]: img import

File: src/pages/UniversalSwap/Component/AssetsTab.tsx [hunk 5]
```diff
@@ -63,6 +63,8 @@ export const AssetsTab: FC<{ networkFilter: string }> = ({ networkFilter }) => {
     }
   ];
 
+  console.log({ listAsset });
```
Comment (quangdz1704) [inline src/pages/UniversalSwap/Component/AssetsTab.tsx]: rm log

**Review by haunv3 — DISMISSED** [review]
lgtm

File: src/pages/UniversalSwap/Component/HeaderTab.tsx [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import { isMobile } from '@walletconnect/browser-utils';
-import ChartImg from 'assets/icons/chart.svg';
-import HideImg from 'assets/icons/show.svg';
-import { ReactComponent as DefaultIcon } from 'assets/icons/tokens.svg';
+import ChartImg from 'assets/icons/chart.svg?react';
+import HideImg from 'assets/icons/show.svg?react';
```
Comment (quangdz1704) [inline src/pages/UniversalSwap/Component/HeaderTab.tsx]: img import

File: src/pages/UniversalSwap/Component/HistoryTab.tsx [hunk 7]
```diff
@@ -1,5 +1,5 @@
-import ArrowImg from 'assets/icons/arrow_right.svg';
-import OpenNewWindowImg from 'assets/icons/open_new_window.svg';
+import ArrowImg from 'assets/icons/arrow_right.svg?react';
+import OpenNewWindowImg from 'assets/icons/open_new_window.svg?react';
```
Comment (quangdz1704) [inline src/pages/UniversalSwap/Component/HistoryTab.tsx]: img import

File: src/pages/UniversalSwap/Component/Routing.tsx [hunk 8]
```diff
@@ -1,6 +1,6 @@
-import { ReactComponent as OraiLightIcon } from 'assets/icons/oraichain_light.svg';
-import { ReactComponent as LineIcon } from 'assets/icons/line.svg';
-import OraiIcon from 'assets/icons/oraichain_light.svg';
+import OraiLightIcon from 'assets/icons/oraichain_light.svg?react';
+import LineIcon from 'assets/icons/line.svg?react';
+import OraiIcon from 'assets/icons/oraichain_light.svg?react';
```
Comment (quangdz1704) [inline src/pages/UniversalSwap/Component/Routing.tsx]: img import

File: src/pages/UniversalSwap/Component/StepByStep.tsx [hunk 9]
```diff
@@ -1,7 +1,7 @@
 import cn from 'classnames/bind';
 import styles from './StepByStep.module.scss';
 import { FC, useState } from 'react';
-import OraiIcon from 'assets/icons/oraichain_light.svg';
+import OraiIcon from 'assets/icons/oraichain_light.svg?react';
```
Comment (quangdz1704) [inline src/pages/UniversalSwap/Component/StepByStep.tsx]: img import

File: src/pages/UniversalSwap/Swap/components/SelectChain/SelectChain.tsx [hunk 10]
```diff
@@ -4,11 +4,11 @@ import styles from './SelectChain.module.scss';
...
 import cn from 'classnames/bind';
 import { chainIcons } from 'config/chainInfos';
-import { ReactComponent as OraiIcon } from 'assets/icons/oraichain.svg';
-import { ReactComponent as IconoirCancel } from 'assets/icons/iconoir_cancel.svg';
+import OraiIcon from 'assets/icons/oraichain.svg?react';
+import IconoirCancel from 'assets/icons/iconoir_cancel.svg?react';
 import { networks } from 'helper';
 import NetworkImg from 'assets/icons/network.svg';
-import CheckImg from 'assets/icons/check.svg';
+import CheckImg from 'assets/icons/check.svg?react';
```
Comment (quangdz1704) [inline src/pages/UniversalSwap/Swap/components/SelectChain/SelectChain.tsx]: img import

File: src/pages/UniversalSwap/Swap/components/TooltipSwapBridge/index.tsx [hunk 11]
```diff
@@ -1,5 +1,5 @@
 import styles from './index.module.scss';
-import ArrowImg from 'assets/icons/arrow_right.svg';
+import ArrowImg from 'assets/icons/arrow_right.svg?react';
```
Comment (quangdz1704) [inline src/pages/UniversalSwap/Swap/components/TooltipSwapBridge/index.tsx]: img import

**Review by haunv3 — DISMISSED** [review]

**Review by quangdz1704 — APPROVED** [review]
look gud to me

**Review by haunv3 — APPROVED** [review]

**Review by quangdz1704 — APPROVED** [review]

**Pull request merged**